recipes-multimedia/gstreamer/gstreamer1.0-plugins-bad_1.16.imx.bb

```diff
@@ -53,118 +47,122 @@ PACKAGECONFIG ??= " \
     bz2 curl dash dtls hls rsvg sbc smoothstreaming sndfile ttml uvch264 webp \
```

Comment:
Should we add `closedcaption` to align with upstream?

Reply:
This would also be added in the next version here, I've left it out since it was not in the old package config.